src/_cffi_src/build_openssl.py

```diff
@@ -112,3 +112,21 @@ def _extra_compile_args(platform):
     libraries=_get_openssl_libraries(sys.platform),
     extra_compile_args=_extra_compile_args(sys.platform),
 )
+
+if __name__ == "__main__":
+    import sysconfig
+    from cffi import recompiler
+
+    module_name, source, source_extension, kwds = ffi._assigned_source
+    c_file = os.path.join(os.path.dirname(os.path.dirname(__file__)), "rust", module_name + source_extension)
+    recompiler.make_c_source(ffi, module_name, source, c_file)
+
+    libraries = _get_openssl_libraries(sys.platform)
+    for lib in libraries:
+        print(f"cargo:rustc-link-lib={lib}")
```

Comment:
I think rust-openssl will also emit `rustc-link-lib` lines, which will (hopefully!) be the same as these. Is this required?